taxi/forms.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.forms import CheckboxSelectMultiple
+
+from .models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number", )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number", )
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=Driver.objects.all(),
+        widget=CheckboxSelectMultiple,

```

Comment:
The `drivers` field in `CarForm` uses `CheckboxSelectMultiple` as its widget. Ensure that this is the intended behavior, as it will allow multiple drivers to be selected for a single car. If the business logic requires only one driver per car, consider using a different field type or widget.